compiler/qsc_rca/src/core.rs

```diff
@@ -2289,6 +2292,60 @@ fn derive_instrinsic_operation_application_generator_set(
     }
 }
 
+fn derive_instrinsic_measurement_application_generator_set(
```

Comment:
Since this function and `derive_instrinsic_operation_application_generator_set` are almost identical, it might be worth just having one function.

Since the information whether the callable is a regular operation or a measurement is in the `callable_context` parameter, I would suggest to use that to populate the runtime features for the inherent compute kind.